core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/QueryResponsePostProcessorImpl.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Codice Foundation
+ * 
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * 
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ * 
+ **/
+package ddf.catalog.impl;
```

Comment:
Since I created this class to avoid adding any more code to the CatalogFrameworkImpl class and make it easier to unit test, I left it in the same package for now, got rid of the interface and renamed the class (see today's commit). The CatalogFrameworkImpl class would need some major refactoring and unit testing so I thought we could always revisit this when we come back to it. Let me know what you think.